src/main/java/ballmerpeak/stargate/proto/ProtoIO.java

```diff
@@ -23,6 +23,7 @@
 	
 	public ProtoIO(Game game) {
 		tileInfo.put(DrawableIndex.FLOOR_WITH_ZPM, "Floor <ZPM>");
+		tileInfo.put(DrawableIndex.FLOOR_WITH_ZPM2, "Floor <ZPM>");
```

Comment:
AFAIK ProtoIO has the same purpose as GameCanvas, even if it implements a different interface.